File: backend/PswProject/PswProject/dto/RegistrationDTO.cs
using PswProject.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PswProject.dto
{
    public class RegistrationDTO
    {
        public int Id { get; set; }
        public String Name { get; set; }
        public String Surname { get; set; }
        public String Username { get; set; }
        public String Password { get; set; }
        public String Phone { get; set; }
        public String Jmbg { get; set; }
        public String Address { get; set; }
        public String Gender { get; set; }
        public Boolean Blocked { get; set; }

        public RegistrationDTO() { }

        public RegistrationDTO(int id, string name, string surname, string username, string password, string phone, string jmbg, string address, string gender, bool blocked)
        {
            Id = id;
            Name = name;
            Surname = surname;
            Username = username;
            Password = password;
            Phone = phone;
            Jmbg = jmbg;
            Address = address;
            Gender = gender;
            Blocked = blocked;
        }
    }
}

File: frontend/psw-frontend/src/app/pages/registration/registration.dto.ts
export class RegistrationDTO{
    Id: number;
    Name: string;
    Surname: string;
    Username: string;
    Password: string;
    Phone: string;
    Jmbg: string;
    Role: number;
    Address: string;
    Gender: string;
    Blocked: boolean;

    constructor(){
        this.Id = 0,
        this.Name = "",
        this.Surname = "",
        this.Username = "",
        this.Password = "",
        this.Phone = "",
        this.Jmbg = "",
        this.Role = 0,
        this.Address = "",
        this.Gender = "",
        this.Blocked = false;
    }
}